00-strings/strings.py

```diff
@@ -0,0 +1,11 @@
+def length(text):
```

Comment:
Should the exercises have [type hints](https://docs.python.org/3/library/typing.html) as I believe is the recommendation nowadays? Of course it's only supported from Python 3.5 onwards, but hasn't 3.5 also reached EOL already?

Reply:
yep, almost year ago and python 3.6 eol is within 4 months. But definitely could be good improvement. This was not used since this direct port from python2 to python3

Reply:
Of course Python is not enforcing typing in any way, but could be nice to show how to "do things right"

Reply:
It improves readability of complex code quite a lot, as well as IDE assistance.